src/gtfs/read.rs

```diff
@@ -1039,29 +1028,23 @@ mod tests {
 
             assert_eq!(2, collections.lines.len());
 
-            let mut lines_ids: Vec<String> = collections
-                .lines
-                .iter()
-                .map(|(_, l)| l.id.clone())
-                .collect();
+            let mut lines_ids: Vec<String> =
+                collections.lines.values().map(|l| l.id.clone()).collect();
             lines_ids.sort();
```

Comment:
This deserve a `fn extract_ids<T: Id<T>>(c: &Collection<T>) -> Vec<String>`

Reply:
absolutely
in another PR?

Reply:
OK